classes/Email.py

```diff
@@ -0,0 +1,8 @@
+from classes.Field import Field
+
+
+class Email(Field):
+    def __init__(self, value):
+        if '@' not in value or '.' not in value.split('@'-1) or ' ' in value:
```

Comment:
Just a suggestion. Maybe we can use a regular expression for validation
r = r'\b[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Z|a-z]{2,}\b'

if (re.fullmatch(r, email)):
        ***
    else:
        raise ValueError('Invalid email address')

Reply:
Good suggestion, I'll add it.